internal/controller/apiextensions/claim/reconciler.go

```diff
@@ -539,7 +555,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 		// We should be watching the composite resource and will have a
 		// request queued if it changes, so no need to requeue.
 		cm.SetConditions(Waiting())
-		return reconcile.Result{}, errors.Wrap(r.client.Status().Update(ctx, cm), errUpdateClaimStatus)
+		return reconcile.Result{}, errors.Wrap(r.client.Status().Update(ctx, cm, r.fieldOwner), errUpdateClaimStatus)
 	}
 
 	record.Event(cm, event.Normal(reasonBind, "Successfully bound composite resource"))
```

Comment:
Not related to this PR, but is this event in correct place? Claim bound to composite before this line and this should rather be something like "Composite is ready" 🤔

Reply:
Agree. Should we open a separate PR to fix that?

Reply:
Given it is a log line I would just do it here, but a separate PR sound good too.